src/components/ProfileContainer.jsx

```diff
@@ -0,0 +1,18 @@
+export default function ProfileContainer() {
+  return (
+    <section>
+      <div className="row gx-3">
+        <div className="col">
+          <div className="card">
+            ProfileRockets
+          </div>
+        </div>
+        <div className="col">
+          <div className="card">
+            ProfileMissions
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+}
```

Comment:
-[x] Well done @Abdelaziz-Ali, for using the JSX component with a very clean and well-structured code.